site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -88,6 +88,7 @@ gcloud container clusters create [CLUSTER_NAME] --cluster-version={{% k8s-versio
   --scopes=gke-default \
   --num-nodes=4 \
   --no-enable-autoupgrade \
+  --enable-image-streaming \
```

Comment:
For changes to the website, we generally gate them so that they do not appear until the next release. Even though this doesn't require new features in Agones, I think it still makes sense to wait to publish this guidance until our next release (which reminds me that we should also recreate the e2e test cluster using your terraform scripts so that our testing starts to use this configuration as well). 

https://agones.dev/site/docs/contribute/#within-a-page describes how to add the feature gate tags to the website pages.

Reply:
Thanks for the context. Fixed it in next commit.
Looks like the line `{{% feature publishVersion="1.27.0" %}}` and `{{% /feature %}}` introduce new lines automatically so I have to add them in the same line with the content to be hidden to avoid messing up the format. Verified via `make site-server` and `make site-server ENV=RELEASE_VERSION=1.26.0 ARGS=`